front/src/modules/ui/modal/components/types/ModalTypes.tsx

```diff
@@ -0,0 +1,2 @@
+export type ModalSize = 'small' | 'medium' | 'large';
```

Comment:
@mustajab-ikram I think you can keep this 2 types directly in the `Modal.tsx` file

Reply:
yes, done